src/frontend/calendar/cell.js

```diff
@@ -25,10 +25,18 @@ function CalendarCell( { blank = false, year, month, day, events } ) {
 				// @todo Get the recurring event time, not just first instance.
 				const eventDate = e.meta.start_date + 'T' + e.meta.time + 'Z';
 				return (
-					<h3 key={ e.id }>
-						{ format( 'g:i a: ', eventDate ) }
-						{ e.title }
-					</h3>
+					<div
+						key={ e.id }
+						role="button"
+						tabIndex={ 0 }
+						onClick={ () => void onEventClick( e ) }
+						onKeyPress={ () => void onEventClick( e ) }
```

Comment:
If you use `Button` (with `isLink` for style), you don't need to define the role, tabindex, or worry about keypresses - we can remove the heading tag in this case too.